.gitignore

```diff
@@ -13,6 +13,7 @@
 # Visual studio temporaries, except a file used by QL4VS
 .vs/*
 !.vs/VSWorkspaceSettings.json
+.vscode/settings.json
```

Comment:
Remove this change

Reply:
Done! Sorry about the multiple forced rebases. Git won't let me drop these files for some reason. Anyways, its all fixed now.